doc/source/getting-started.rst

```diff
@@ -0,0 +1,196 @@
+.. _getting-started:
+
+*********************
+Getting Started Guide
+*********************
+
+The following guide will step through an example of developing and testing a
+new Ansible role. After reading this guide, you should be familiar with the
+basics of how to use Molecule and what it can offer.
+
+.. note::
+
+    In order to complete this guide by hand, you will need to additionally
+    install `Docker`_. Molecule requires an external Python dependency for the
+    Docker driver which is provided if you install Molecule using ``pip install
+    'molecule[docker]'``. Otherwise, you can ``pip install docker``.
+
+.. _Docker: https://docs.docker.com/
+
```

Comment:
I wonder if a bullet point overview of the rest of the page here would help, give some content for what people will read as well as giving high level context/

Reply:
Solid idea! `toctree` to the rescue ...